lite/examples/image_classification/ios/ImageClassification/Camera Feed/CameraFeedManager.swift

```diff
@@ -15,6 +15,10 @@
 import UIKit
 import AVFoundation
 
+#if targetEnvironment(simulator)
```

Comment:
I believe having many `#if` conditions would hurt readability, which is not ideal for an example app. Let's try to avoid using them.

In this particular case, looks like this `#if` statement would not be necessary at all.
In the simulator version, can't we just not use this `CameraFeedManagerDelegate`?

Reply:
Removed most of the `#if` statements, but kept two around starting and stopping the camera session. They are much smaller than the previous ones and shouldn't impact readability as much (since both branches are clearly visible).